internal/cephfs/volumeoptions.go

```diff
@@ -48,6 +48,32 @@ type volumeOptions struct {
 	FuseMountOptions    string `json:"fuseMountOptions"`
 	SubvolumeGroup      string
 	Features            []string
+
+	// conn is a connection to the Ceph cluster obtained from a ConnPool
+	conn *util.ClusterConnection
+}
+
+// Connect an CephFS volume to the Ceph cluster.
+func (vo *volumeOptions) Connect(cr *util.Credentials) error {
+	if vo.conn != nil {
+		return nil
+	}
+
+	conn := &util.ClusterConnection{}
+	if err := conn.Connect(vo.Monitors, cr); err != nil {
+		return err
+	}
+
+	vo.conn = conn
+	return nil
+}
+
+// Destroy cleans up the CephFS volume and closes the connection to the Ceph
```

Comment:
This won't clean up the CephFS volume right, it will just close the connection

Reply:
Yes, but we might free other resources later, we use `Destroy()` in other places too.

Reply:
IMO cephfs volume will be created when you create a filesystem in ceph. as cephfs cannot clean up or create filesystem we need to update the comment to match it.

Reply:
We use the same with `rbdVolume.Destroy()`. It is a destruction of the object, not the removal of a volume. Do you have a suggestion for a name of a method that frees resources, but not the backing storage?

Reply:
I don't have any objection with function name, my comment was mainly related to function comment, I think that should be finr